pythoscope/serializer.py

```diff
@@ -278,8 +278,13 @@ def __init__(self, obj, serialize):
             self.constructor_format = self.type_formats_with_imports[type(obj)][0]
             self.imports = self.type_formats_with_imports[type(obj)][1]
 
-    # def __repr__(self):
-    #     return "SequenceObject(%s)" % (self.constructor_format % self.contained_objects)
+#    def __eq__(self, other):
+#        return False
+           
+            
+#    def __repr__(self):
+#        import ipdb; ipdb.set_trace()
+#        return "SequenceObject(%s)" % (self.constructor_format % self.contained_objects)
```

Comment:
Please don't add more dead code. This `__repr__` was probably commented out by accident.